src/tests/unit/tests/electron/window-management/window-frame-updater.test.ts

```diff
@@ -32,6 +32,12 @@ describe(WindowFrameUpdater, () => {
             testSubject.initialize();
         });
 
+        it('invokes enterFullScreen', () => {
+            ipcRendererShimMock.setup(b => b.maximizeWindow()).verifiable(Times.once());
+
+            windowFrameActions.maximize.invoke(null);
```

Comment:
Based on the test name, it looks like this should say enterFullScreen instead of maximizeWindow/maximize

Reply:
Fixed copy/paste error